005-RockPaperScissorsBoom/Student/Guides/challenges/Setup.md

```diff
@@ -13,53 +13,53 @@
 
 The first challenge is to setup an environment that will help you build the Rock Paper Scissors Boom! Game Server and deploy it locally. We need to make sure everything is working before bringing it to Azure.
 
- To *really* do this locally, you would need the app platform (.NET Core, SQL Server, IIS, etc) running locally. We probably do not have enough time for that. Enter Azure! We can set up server in Azure to replicate a local environment where we can build our application and test deploying it. All we need running on that server is Docker and we can leverage containers for building and deploying the application; even hosting the database!
- 
+ To *really* do this locally, you would need the app platform (.NET Core, SQL Server, IIS, etc) running locally. We probably do not have enough time for that. Wouldn't be nice to have a cloud-based container image building for platforms including Linux, Windows, and ARM?. Enter Azure! We can leverage on demand services to extend your "inner-loop" development cycle to provide container image build, push and runs!
+
  But, even in that case, we need some tools locally to manage Azure and that can take a while to setup. The good news is that by using the Azure Shell - we have all the tools we need.  All we really need is a browser and an Azure subscription.
 
-> The use of Azure Cloud Shell is our recommendation to simplify your experience. But if you prefer using your local machine to do the lab, feel free to do it. Based on our experience, the setup of Docker on local machine (which is required for the challenges) could take some time and could have some issues.*
+> The use of Azure Cloud Shell is our recommendation to simplify your experience. But if you prefer using your local machine to do the lab, feel free to do it. You would need either the [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli) or [Azure PowerShell](https://docs.microsoft.com/powershell/azure/install-az-ps?view=azps-5.4.0).
 
  **Setting up this mock *local* environment is important. We will be using this environment for future challenges.**
 
 ## Challenges
 
 1. [Fork](https://help.github.com/articles/fork-a-repo/) this repo in your GitHub account. We want you to create your own fork so that if you have suggestions or fixes, you can issue a pull-request and submit them back to the main repo.
```

Comment:
As per my overall comment on the PR, this is where I would modify the instructions to refer to obtaining the code from the Resource package (zip file) provided by the coach instead of forking and cloning this repo.

Reply:
Hey Romer, sorry I've left this WTH in a state that's not completely compatible with the new WTH standards. Thank you for updating it!

Reply:
@jrzyshr yes, that makes sense and agree. I think i ll need that for the session so i ll make that a priority.

Reply:
i thinks this is addressed with latest changes and workflow